CoreMapView/src/au/gov/asd/tac/constellation/views/mapview2/layers/ThiessenPolygonsLayer.java

```diff
@@ -674,18 +652,43 @@ private void connectMarkersToCorners() {
             } else if (!relevantMarkerIds.isEmpty()) {
                 n.addRelevantMarker(relevantMarkerIds.get(0));
             }
+        }
+    }
 
+    /*private void findCornerNeighbours(IntersectionNode nearest, IntersectionNode parent, IntersectionNode grandParent) {
```

Comment:
is the intention to come back to this function? If not, can be removed

Reply:
Yes this could be a solution to get rid of some duplicate code